Formula/gambit-scheme.rb

```diff
@@ -42,17 +41,11 @@ def install
     ]
 
     system "./configure", *args
-
-    # Fixed in gambit HEAD, but they haven't cut a release
-    inreplace "config.status" do |s|
-      s.gsub! %r{/usr/local/opt/openssl(?!@1\.1)}, "/usr/local/opt/openssl@3"
-      s.gsub! %r{/usr/local/opt/openssl(@\d(\.\d)?)?}, Formula["openssl@3"].opt_prefix
-    end
```

Comment:
Might need to keep some of this.

Reply:
Could be done on `configure` instead..

Reply:
configure --with-openssl=/usr/local/myssl works.